log.go

```diff
@@ -111,7 +111,7 @@ func Infof(format string, args ...interface{}) {
 
 func (gl *grouplogger) Infof(format string, args ...interface{}) {
 	for _, logger := range gl.loggers {
-		writeMessage(logger, 1, SeverityInfo, format, args...)
+		writeMessage(logger, 2, SeverityInfo, format, args...)
```

Comment:
You already figure that out, so I am answering this for the history. The previous commit introduced one more level from log package before this one, therefore we need to increment this number to account for that extra function call.